### PR TITLE
[WIP] Update xlnet-large glue results

### PR DESCRIPTION
**Pull request by zburning**
## Description ##
Update xlnet-large glue results & small changes to the run_glue.py scripts.

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1104/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1104/1/index.html

File: scripts/language_model/run_glue.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 from model.XLNet_classifier import XLNetClassifier
 from transformer import model
 
+os.environ['MXNET_USE_FUSION'] = '0'
```
Comment (leezu) [inline scripts/language_model/run_glue.py]: This shouldn't be needed anymore

File: scripts/language_model/run_glue.py [hunk 2]
```diff
@@ -43,75 +44,148 @@
...
+
+
+def split_and_load(arrs, _ctxs):
     """split and load arrays to a list of contexts"""
     assert isinstance(arrs, (list, tuple))
     # split and load
-    loaded_arrs = [mx.gluon.utils.split_and_load(arr, ctx, even_split=False) for arr in arrs]
+    loaded_arrs = [[
+        i.as_in_context(ctx)
+        for i, ctx in zip(split_array(arr, len(_ctxs)), _ctxs)
```
Comment (leezu) [inline scripts/language_model/run_glue.py]: This is a workaround for MXNet 1.6, as https://github.com/apache/incubator-mxnet/pull/17123 is only available in MXNet 1.7?

Could you add a comment like "TODO Replace with ... once targeting MXNet 1.7"

**Comment by mli** [comment]
Job PR-1104/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1104/3/index.html

**Pull request merged**